Dockerfile

```diff
@@ -68,3 +68,5 @@ COPY --from=busybox-min --chown=100:1000 /home/relayer /home/relayer
 
 WORKDIR /home/relayer
 USER relayer
+
+ENTRYPOINT ["rly", "start"]
```

Comment:
`ENTRYPOINT ["rly", "start"]` (like Gaia) or `ENTRYPOINT ["rly"]` (like IBC-Go)

Reply:
going with the gaia approach, does not matter since it can be overridden if the user decides too. default just starts